store/metric.go

```diff
@@ -131,6 +132,38 @@ func (m *mongoDB) CollectRawMetrics(location schema.Location) (*schema.Metric, e
 	}, nil
 }
 
+// SyncProfileIndividualMetrics calculate individual metrics and save into profile
+func (m *mongoDB) SyncProfileIndividualMetrics(profileID string) (*schema.IndividualMetric, error) {
+	now := time.Now().UTC()
+
+	symptomsToday, symptomsYesterday, err := m.GetSymptomCount(profileID, nil, 0, now)
```

Comment:
Neighborhood metrics only considers today's data. I thought the time range of individual metrics would be the same.

Reply:
I do not understand. It use now to calculate today's data. Yesterday's data is to calculate the delta.

Reply:
My bad. Your usage is right. I mistook this function for another.

Reply:
I only thing I think we need to improve is the time period. It is using UTC time instead of local time. Since you are working on the trend function, I think you could review the function and extend it by giving an time period parameter, for example, started_at. And take the timezone into consideration.

I reviewed the function, it's like we can set timezone when using `$dateToString`.
https://docs.mongodb.com/manual/reference/operator/aggregation/dateToString/

Reply:
That sounds good. I might add one more param `period` so the aggregation can be based on different time intervals.

Reply:
My future PR will also handle the changes here once I modify `GetSymptomCount` and `GetBehaviorCount`.

Reply:
OK